TypeCobol.Test/Parser/Programs/Cobol85/ProgramEndMismatchStacked1.rdz.cbl

```diff
@@ -0,0 +1,19 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Main.
+      
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Nested0.
+      
+       END PROGRAM Nested0.
+      
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Nested1.
+      
+       END PROGRAM Nested1.
+      
+       END PROGRAM Main.
+      
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Stacked1.
+      
+       END PROGRAM Toto.
```

Comment:
Maybe I misread the specification, but it seemed that for the last program of a sequence, a missing 'END PROGRAM' should not generate any error : 

- Exception for the last program in a sequence of source programs, then don't generate an error (IBM doc)

Reply:
But in fact, I think it's better this way, so ok.

Reply:
IBM's compiler generates the same error here, because the program has an `END PROGRAM` but it has the wrong program name.